inductiva/users/methods.py

```diff
@@ -0,0 +1,39 @@
+"""Methods to interact with the user info on the API."""
+import json
+from typing import Any, Dict, List
+
+import inductiva
+from inductiva import api
+from inductiva.client import ApiClient, ApiException
+from inductiva.client.apis.tags.users_api import UsersApi
+
+
+def _fetch_quotas_from_api() -> List[Dict]:
+    """Get information about a user's quotas.
+    """
+    api_config = api.validate_api_key(inductiva.api_key)
+
+    with ApiClient(api_config) as client:
+        api_instance = UsersApi(client)
+        try:
+            resp = api_instance.get_user_quotas().response
+
+            response_body = json.loads(resp.data.decode("utf-8"))
+
+            return response_body
+
+        except ApiException as e:
+            raise e
+
+
+def get_quotas() -> Dict[str, Dict[str, Any]]:
```

Comment:
this method is basically a wrapper for `_fetch_quoatas_from_api`. Why not have a single `get_quotas` method?